src/pieces.cpp

```diff
@@ -91,6 +91,8 @@ void Piece::set() {
 	board.setPiece(rows, cols, this->color);
 }
 
+void Piece::draw() {}
```

Comment:
assuming this purposely a stub for now. I suggest you add TODOs to make reviewing this easier for me.